gremlin-go/driver/client.go

```diff
@@ -41,7 +41,7 @@ type Client struct {
 }
 
 // NewClient creates a Client and configures it with the given parameters.
```

Comment:
Probably also want to call out that it also creates the connection (or attempts to) and will re-turn a non-nil error if creating this connection fails.